modules/basics/src/main/resources/com/opengamma/strata/config/base/FloatingRateNameData.ini

```diff
@@ -89,6 +89,8 @@ USD-FEDFUND-H.15-OIS-COMPOUND = USD-FED-FUND
 CAD-CORRA = CAD-CORRA
 CAD-CORRA-OIS-COMPOUND = CAD-CORRA
 
+AUD-AONIA = AUD-AONIA
```

Comment:
Need to add to two sections. Under the [ibor] section:

```
AUD-BBSW = AUD-BBSW
AUD-BBR-AUBBSW = AUD-BBSW
AUD-BBR-BBSW = AUD-BBSW
AUD-BBR-BBSW-Bloomberg = AUD-BBSW
AUD-BBR-BBSW-Bloomberg = AUD-BBSW
```

and under [overnightCompounded] section:

```
AUD-AONIA = AUD-AONIA
AUD-AONIA-OIS-COMPOUND = AUD-AONIA
```